docs/contributing.md

```diff
@@ -1,49 +1,85 @@
 # Developer docs
 
-## Contributing to JAX AI Stack Documentation
+## Contributing to the JAX AI Stack Documentation
+
 The documentation in the `jax-ai-stack` repository is meant to build on documentation
 of individual packages, and specifically cover topics that touch on multiple packages
 in the stack. If you see something missing and would like to contribute, please first
 [open an issue](https://github.com/jax-ml/jax-ai-stack/issues/new) and let us know what
 you have in mind!
 
+## Pre-requisites
+
+To contribute to the documentation, you will need to set your development environment.
+
+You can create a virtual environment and install the dependencies with the following commands:
+
+```bash
+virtualenv jax-docs
+# activate the virtual environment
+source jax-docs/bin/activate
+# install the dependencies
+python -m pip install -r docs/requirements.txt
+```
```

Comment:
I'd prefer not to document any approach to defining the environment, because that's not really a concern of this package, and we have to good reason to recommend one appraoch over another. We could say in words "create a virtual environment or conda environment" and leave it at that – what do you think?